TypeInjections/Tool/Tool.fs

```diff
@@ -7,12 +7,48 @@ open LibGit2Sharp
 open System.IO
 open System
 open TypeInjections
+open CommandLine
 
 module Tool =
+    
+    
+    [<Verb("local", HelpText = "[old project path] [new project path] [outputPath]")>]
+    type LocalOptions =
+        { [<Option('o', "old", Required = false, HelpText = "folder path of old project")>]
+          oldFolder: string
+          [<Option('n', "new", Required = false, HelpText = "folder path of new project")>]
+          newFolder: string
+          [<Option('p', "output", Required = false, HelpText = "folder path to place typeCart files")>]
+          outputFolder: string
+          [<Option('f', "file", Required = false, HelpText = "specify filename you want to run typecart on")>]
+          file: seq<string> }
+
+    [<Verb("remote", HelpText = "[github repo url] [new commit] [old commit] [outputPath]")>]
+    type RemoteOptions =
```

Comment:
I think remote should just not exist, at least not as a verb. Let's add an optional `--clone <repo url>` argument to the `git` verb and if it's provided we clone the repository first.

Reply:
I really like this idea, included!